images/build/benv.sh

```diff
@@ -216,18 +216,17 @@ benv-resolve() {
       return 1
     fi
 
+    IFS='.' read -ra SPLIT_VERSION <<< "$value"
+    majorAndMinorParts="${SPLIT_VERSION[0]}.${SPLIT_VERSION[1]}"
+
     export LD_LIBRARY_PATH="$platformDir/lib:$LD_LIBRARY_PATH"
 
-    local DIR="$platformDir/bin"
-    updatePath "$DIR"
-    if [ -e "$DIR/python2" ]; then
```

Comment:
Made it simpler here. For reference, following is how a Python runtime folder looks like:

![image](https://user-images.githubusercontent.com/6559784/88946993-c69cdf00-d244-11ea-8fc3-72783f2a80a9.png)